packages/algoliasearch-helper/test/spec/hierarchical-facets/with-a-disjunctive-facet.js

```diff
@@ -29,7 +29,7 @@ test('hierarchical facets: combined with a disjunctive facet', function () {
 
   helper.setQuery('a').search();
 
-  var disjunctiveFacetsValuesQuery = client.search.mock.calls[0][0][1];
+  var disjunctiveFacetsValuesQuery = client.search.mock.calls[0][0][2];
```

Comment:
Was that wrong before?

Reply:
the order of queries has changed (is now ordered by attribute)